src/features/footer/components/FooterItem.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+const NUMBER_FORMATTER = new Intl.NumberFormat();
+const getSign = (value: number) => (value > 0 ? "positive" : "negative");
+
+const signFormat = (value: number) => {
+  return getSign(value);
+};
+
+type FooterData = {
+  exchange: string;
+  price: number;
+  priceChange: number;
+  percentageChange: number;
+};
+
+export const FooterItem = ({
```

Comment:
You used `FC<Props>` in the component above -- which I prefer personally because it ensures you return the right kind of thing